adb_graphics/figures/maps.py

```diff
@@ -300,12 +302,29 @@ def draw(self, show=False):
         # Levels should be included in the settings dict here since they don't
         # correspond to a full field of contours.
         if self.hatch_fields:
+            handles = []
             for field in self.hatch_fields:
-                self._draw_field(ax=ax,
-                                 field=field,
-                                 func=self.map.m.contourf,
-                                 **field.contour_kwargs,
-                                 )
+                colors = field.contour_kwargs.get('colors', 'k')
```

Comment:
There is a type mismatch in colors here. If you don't find a 'colors' key (list) in the dict, you get back a string by default.